### ISSUE TITLE
Фотографии некоторых объектов повёрнуты в браузере Safari

### ISSUE THREAD

**Issue by evgenykatyshev** [heritagemap/frontend-pwa#169]
Пользователи сообщили, что фотографии некоторых объектов повёрнуты при просмотре в Safari на Mac. Пока единичный пример, на скрине:

![image](https://user-images.githubusercontent.com/21023515/99794987-1a0ca300-2b3c-11eb-97df-321ac6459519.png)

**Comment by batyrmastyr** [comment]
Похоже, проблема в изображении: оно снято повёрнутым, но в [EXIF](https://magnus-toolserver.toolforge.org/commonsapi.php?image=Houserambler.jpg&meta)  прописано как его нужно повернуть — ``<meta><item name="Orientation">6</item>``.
Проблема должна быть [во многих браузерах](https://caniuse.com/css-image-orientation):
-  IE/Edge < 81 
- Хром < 81 (вышел в апреле этого года)
- Сафари для MacOS X < 13 версии, вышел в марте этого года
Проблемы нет в мобильном Сафари и Firefox - там фотки поворачиваются с 2010 и 2013 года соответственно.

Тут два варианта:
1.  скачивать фотки яваскриптом и поворачивать вручную. По советам со [StackOverflow](https://stackoverflow.com/questions/20600800/js-client-side-exif-orientation-rotate-and-mirror-jpeg-images):
1.1. Взять [blueimp/JavaScript-Load-Image](https://github.com/blueimp/JavaScript-Load-Image). 25 Кб в самом сжатом виде, но можно вырезать кучу ненужных функций (масштабирование, вырезание части изображения и т.д.).
1.2. Использовать canvas.DrawImage и использовать более простое определение ориентации:  [1](https://stackoverflow.com/a/64037903), [2](https://stackoverflow.com/a/46814952) и [3](https://gist.github.com/mindplay-dk/72f47c1a570e870a375bd3dbcb9328fb). Кто-то печалился, что с отзывчивостью у холста дела хуже, чем у тега img, возможно речь шла про замыливание на экранах с высоким разрешением.
1.3. пункт 1.2 только ориентацию брать из api.
1.4. грязный хак ``<iframe src="/my-image.jpg"></iframe>`` из [этого](https://stackoverflow.com/a/39384061) ответа.
1.5. пункты 1.2 - 1.4 применять только если браузер не поворачивает картинку сам. Код взять [там](https://github.com/blueimp/JavaScript-Load-Image/blob/master/js/load-image-orientation.js#L58-L104).
2. прогонять фотки через свой сервер и переворачивать / обрезать их там. По идее самый правильный в перспективе вариант: можно будет не так сильно зависеть от надёжности чужих api.

**Comment by avdeev** [comment]
@batyrmastyr Да, думаю, надо попробовать второй вариант. Можно в сторону вот этих проектов посмотреть.
https://github.com/imgproxy/imgproxy
https://github.com/willnorris/imageproxy

**Comment by Julia-Lavrova** [comment]
https://heritagemap.ru/lat/56.317176/lon/43.988198/5200615000
